frontend/public/components/masthead-toolbar.jsx

```diff
@@ -600,12 +601,13 @@ class MastheadToolbarContents_ extends React.Component {
           </PageHeaderToolsGroup>
           <PageHeaderToolsGroup>
             {/* mobile -- (notification drawer button) */
-            alertAccess && !notificationsRead && (
+            alertAccess && notificationAlerts?.data?.length > 0 && (
               <PageHeaderToolsItem className="visible-xs-block">
```

Comment:
Hi @jcaianirh, does this mean if there are no notifications the badge won't show up in the masthead?  Is that what we want? -thanks

Reply:
nvm, I see this is for mobile.  :+1: